docs/content/docs/connectors/table/formats/avro-confluent.md

```diff
@@ -287,6 +287,14 @@ Format Options
             <td>String</td>
             <td>The URL of the Confluent Schema Registry to fetch/register schemas.</td>
         </tr>
+        <tr>
+            <td><h5>auto.register.schemas</h5></td>
+            <td>optional</td>
+            <td>yes</td>
```

Comment:
why is this line here but not in the Chinese?

Reply:
I had assumed we would just add here as the optional above is not translated.  `<td>yes</td>` as per the note in the link you sent. But it wounds like you have this in hand